src/index.css

```diff
@@ -3,17 +3,18 @@ body {
   font-family:
```

Comment:
- [ ] As the previous reviewer suggests kindly make sure your app matches the Zeplin design:
   - [ ] Kindly make sure to add the progress circle to the book card
   - [ ] Also make sure the `add new book` form is styled exactly as in the zeplin design
   - [ ] The book cards don't have a border in the provided design so it should not be included in the app
   - [ ] Kindly make sure to use the exact fonts from the design in your app and style the `Remove` button/link in the book cards as the other buttons (`Comment` and `Edit`)

|Your app|Zeplin design|
|-|-|
|![Screenshot_20230312_000821](https://user-images.githubusercontent.com/25227022/224515424-70d01425-7da7-410c-b17b-35ed6d86e964.png)|![image](https://user-images.githubusercontent.com/25227022/224515427-265e49ab-de58-4b29-aaa5-d2e5f64140a9.png)|